CppSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.h

```diff
@@ -111,6 +112,7 @@ class CreateLoadReport : public Esri::ArcGISRuntime::Authentication::ArcGISAuthe
   QString m_serviceCategoryName;
   QString m_terminalName;
   QString m_tierName;
+  QUrl m_featureLayerUrl;
```

Comment:
this should be const and initialized in the constructor initializer list. 
```suggestion
  const QUrl m_featureLayerUrl;
```

Reply:
Updated